docs/integrations/sources/paypal-transaction.md

```diff
@@ -87,8 +87,9 @@ Transactions sync is performed with default `stream_slice_period` = 1 day, it me
 
 | Version | Date       | Pull Request                                             | Subject                                                                                                                      |
 |:--------|:-----------|:---------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------|
-| 2.1.1 | 2023-10-19 | [31599](https://github.com/airbytehq/airbyte/pull/31599) | Base image migration: remove Dockerfile and use the python-connector-base image |
-| 2.1.0   | 2023-08-14 | [29223](https://github.com/airbytehq/airbyte/pull/29223) | Migrate Python CDK to Low Code schema                                                                                                     |
+| 2.1.2 | 2023-10-23 | [31759](https://github.com/airbytehq/airbyte/pull/31759) | Keep transaction_id as a string and fetch data in 7-day batches                                                              |
```

Comment:
this is not a breaking change from 2.1.0, but this is a breaking change from 2.0.0. Considering `2.1.0` is broken, is it worth bumping to `3.0.0`?

Reply:
we'll also need to update the connection state on cloud because it's pinned to `2.0.0`

Reply:
> Considering 2.1.0 is broken, is it worth bumping to 3.0.0?

I'm not sure. I think it would be more fitting to have marked 2.1.0 as the breaking change 🤔 To be clear, the change to state makes it breaking in the "can't roll back" sense, but the roll forward went okay, right?

Reply:
ah right! the roll forward was fine so no big deal here 👍 

> retrospectively mark 2.1.0 as the breaking change
Is this just a matter of updating the metadata file with a breaking change block?

Reply:
> Is this just a matter of updating the metadata file with a breaking change block?

It is! 

In this case I believe the cloud users would see the banner, since they're currently back on 2.0.0, but I think we can opt in for them? 🤔 Alternatively we could wait until after this PR to mark 2.1.0 as breaking

Reply:
yep, that's fine!